### ISSUE TITLE
Google docs table navigation: NVDA reads the whole row when using up and down arrow in Firefox

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#9571]
### Steps to reproduce:
1. Open a new document in google docs
2. Enable screen reader and also Braille suport
3. Create a new table with 3 rows and 3 columns
4. Place the carret in the first cell
5. Press down arrow

### Actual behavior:
NVDA will read "row 2 column 1, column 2. column 3. While I understand the desire for reading the whole row in some cases, this should be optional if possible.

### Expected behavior:
NVDA should read row 2 column 1

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
alpha-17240,c20a5032

#### Windows version:
Win10 1809 Update

#### Name and version of other software in use when reproducing the issue:
Firefox 66.0.5

#### Does the issue still occur after restarting your PC?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, in 2019.1.1 and 2018.4.1 it is the same issue.

**Comment by Adriani90** [comment]
cc: @jcsteh, @leonardder mybe I am wrong and this is an issue in Google docs rather than NVDA but I don't really know.

**Comment by jcsteh** [comment]
This is somewhat controversial. In general, I'd agree; it's consistent with what we do elsewhere. However, in terms of what Firefox treats as a line, it's a little trickier because it really is on the same line as far as the up/down arrow keys are concerned, which is generally how "line" is determined. I'm inclined to treat it as a bug, but I need to give it some more thought.

**Comment by Adriani90** [comment]
@jcsteh I actually don't think that this is a bug perse, but Maybe it could be made optional to read the whole row or to read the current cell only. At least in other applications like MS Word for example, NVDA reads only the current cell when pressing down arrow in tables.

**Comment by Adriani90** [comment]
Actually this behavior is just fine and if optional, it could be extended to other browsers and programs with tables as well.

**Comment by jcsteh** [comment]
I can't reproduce this with the cache (now default). When I use the up and down arrow keys in a table, only the current cell is read, which is consistent with what NVDA does elsewhere.

**Comment by Adriani90** [comment]
Ah right, with cache enabled this seems consistent like in other browsers. In this case I am closing this issue and will probably create a new feature request with more details reg. globally read entire row when navigating in first column row by row.
